runtests.py

```diff
@@ -2118,6 +2132,9 @@ def main():
     if options.xml_output_dir:
         shutil.rmtree(options.xml_output_dir, ignore_errors=True)
 
+    interval = 10
+    if not options.capture:
+        interval = 10000
```

Comment:
Perhaps set this to None and modify time_stamper_thread to return an empty context in that case?

Reply:
fixed